ai/Gemini_diary_summary.py

```
@@ -0,0 +1,69 @@
+    except FileNotFoundError:
+        print("대화 기록 파일을 찾을 수 없습니다.")
+        return None
+
+def save_summary(summary):
+    today = datetime.now().strftime("%Y-%m-%d")
+    filename = f"summary_{today}.txt"
+    try:
+        with open(filename, 'w', encoding='utf-8') as f:
+            f.write(summary)
+        print(f"일기가 {filename}에 저장되었습니다.")
+    except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)